ci/.env

```diff
@@ -3,8 +3,29 @@
 # Choose the CI you want to run the deployments.
 # Both CI will make tests but only the one specified will deploy.
 # Current available choices are travis and circleci.
-export CURRENT_CI='travis'
-export MULTI_BRANCH=0
-export REPOSITORY='api-platform/demo'
-export DEPLOYMENT_BRANCH='master'
-export PROD_DNS='demo.api-platform.com'
+export CURRENT_CI=travis
+
+# If you want to deploy feature branches, set FEATURE_DEPLOY value to 1.
```

Comment:
It's already the case, should we say "If you doesn't want to deploy feature branches, set FEATURE_DEPLOY value to 0." instead ?